### PR TITLE
edited install

### PR DESCRIPTION
**Pull request by annbech**
*Description of changes:* Editorial review of install.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by aaronmarkham** [comment]
Ok, this is better, but we're still getting the `^M` stuff in there. Can you try out [this Line Ending Selector package](https://github.com/atom/line-ending-selector) for Atom and adjust the settings so that you use Unix style carriage returns? It should be set to `LF`.

**Comment by annbech** [comment]
Great! Thanks. Onward.

File: docs/README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
-# MXNet Model Server Documentation
+# Model Server or Apache MXNet Documentation
 
 ## Basic Features
-* [Serving Quick Start](../README.md#serve-a-model) - Basic server usage tutorial
-* [Exporting Quick Start](../README.md#export-a-model) - Basic export usage tutorial
-* [Installation](install.md) - Installation and troubleshooting
-* [Serving Models](server.md) - How to use `mxnet-model-server`
+* [Serving Quick Start](../README.md#serve-a-model)_- Basic server usage tutorial
```
Comment (lupesko) [inline docs/README.md]: nit: is this underscore needed?

File: docs/install.md [hunk 2]
```diff
@@ -1,67 +1,54 @@
...
 
 ## Prerequisites
 
-* **python**: required; MMS works with Python 2 or 3; you may also want to use a Python/Conda environment when installing MMS, so it doesn't conflict with your other MXNet or ONNX installations.
-* **protoc**: optional; if you plan to use the ONNX features, you will need to have the [protobuf compiler installed](https://github.com/onnx/onnx#installation). Install this *before* installing MMS.
-* **curl**: optional; it's used in all of the examples, so install it via your package manager of choice.
-* **unzip**: optional; it's nice to have and to associate with `.model` extensions, so you can easily extract model files and inspect their contents.
+* **Python**: Required. Model Server for Apache MXNet (MMS) works with Python 2 or 3.  When installing MMS, we recommend that you use a Python and Conda environment to avoid conflicts with your other Apache MXNet or Open Neural Network Exchange (ONNX) installations.
+
* **protoc**: Optional. If you plan to use ONNX features, you need to install the [protobuf compiler](https://github.com/onnx/onnx#installation). Install it *before* installing MMS.
```
Comment (lupesko) [inline docs/install.md]: What are all these ^M? (more below...)

File: docs/README.md (hunk 1 above)
Comment (lupesko) [inline docs/README.md]: nit: unnecessary underscore?

**Comment by lupesko** [comment]
@aaronmarkham can you please follow up on the requested changes? this is becoming pretty stale...

**Comment by aaronmarkham** [comment]
I merged #332 which seems to have dealt with the characters so we can close this one. 
Ann, please provide further PRs for your edits. They should go through the normal process now that you have a text editor that doesn't load up those windows formatted line breaks, etc.
Thank you!

**Pull request closed without merging**